skills/skill-parallel-agents/SKILL.md

```
@@ -226,13 +226,11 @@ If the detect-providers command shows BOTH providers are missing (CODEX_STATUS=m
 > Or set API key: `export OPENAI_API_KEY="sk-..."`
 > Get key from: https://platform.openai.com/api-keys
 >
-> **Option 2: Google Gemini** (best for analysis)
+> **Option 2: Google Antigravity CLI** (best for analysis)
 > ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)